docker/generic/start_proxy.py

```diff
@@ -456,15 +456,21 @@ def make_argparser():
         help="An alias to override --tracing_sample_rate to 0")
     parser.add_argument(
         '--tracing_incoming_context',
-        default="",
+        default="traceparent",
         help='''
-        comma separated incoming trace contexts (traceparent|grpc-trace-bin|x-cloud-trace-context)'''
+        Comma separated incoming trace contexts (traceparent|grpc-trace-bin|x-cloud-trace-context).
+        
+        See official documentation for more details:
+        https://cloud.google.com/endpoints/docs/openapi/tracing'''
```

Comment:
This page is for ESPv1, we need to add some sections for ESPv2.  Could you file a bug to track this task?

Reply:
Yup, I am working on a draft right now. b/172831233 tracks it.